orthoinference/src/main/java/org/reactome/orthoinference/SkipTests.java

```diff
@@ -0,0 +1,219 @@
+package org.reactome.orthoinference;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+import org.gk.model.ClassAttributeFollowingInstruction;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceUtilities;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+
+public class SkipTests {
```

Comment:
Is this code for testing? If so, it should probably be moved from src/main/java/... to src/test/java/...

Reply:
They are tests that determine if an instance should have an inference run it or not, effectively testing if they should be skipped

Reply:
Ok, never mind.